denizen_scripts/survival/repo_link/MobScripts/CreeperStuff.dsc

```diff
@@ -4,4 +4,10 @@ Creeper_Prime_Signal:
   events:
     on entity explosion primes:
       - If <context.entity.is_mythicmob>:
-        - mythicsignal <context.entity> PrimetimeShowtime source:<context.entity>
+        - mythicsignal <context.entity.mythicmob> PrimetimeShowtime source:<context.entity>
+
+#creeper_vents:
+#Placeholder to come later.
+
+#creeper_splosion:
+#Placeholder to come later.
```

Comment:
ideally, instead of placing placeholders in our scripts, we should open an issue or comment alongside the pull request for a future issue to create the placeholder script; a to-do is better left someplace available for a conversation/further description rather than within one of (currently) 606 scripts
```suggestion
```

Reply:
Was taking with Xeane on how to do it and forgot to remove them before submission.